wordpress-mobile/WordPress-iOS
# Nuking SignupViewController

### Details:
In this PR we're effectively removing **SignupViewController**, which has been replaced by the new Signup Flow, and is not in use anymore.

Ref. #8109

@elibud @ScoutHarris @nheagy mind taking a peek at this one?

Thanks in advance!!

🔥 🔥 🔥 🔥 🔥 🔥 


### To test:
1. Verify the unit tests run correctly
2. Verify that `Signup with Email` works as expected
3. Verify that `Signup with Google` works as expected
4. Log into a Self Hosted. Open > Me > Log In > Signup, and verify the slider shows up correctly.
